highfidelity/hifi
# Modal dialogs async

case: https://highfidelity.fogbugz.com/f/cases/6821/Modal-dialog-s-blocks-the-thread-that-creates-them
The PR is for preliminary understanding of things gets right

Test plan:
- Goto Toolbar mode
- Try to do activities with MessageBox dialogs gets opened. Mostly within Marketplace
- Goto Tablet mode
- make sure everything still works
- Test file open dialogs: in Running Scripts try to load scripts from disk
- Test input fields: in Running Scripts try to load scripts from URL
